tests/bearlib/abstractions/LinterTest.py

```diff
@@ -144,18 +144,20 @@ def test_decorator_invalid_states(self):
 
     def test_decorator_generated_default_interface(self):
         uut = linter('some-executable')(self.ManualProcessingTestLinter)
-        with self.assertRaises(NotImplementedError):
+        with self.assertRaisesRegex(NotImplementedError, ''):
             uut.create_arguments('filename', 'content', None)
 
     def test_decorator_invalid_parameter_types(self):
         # Provide some invalid severity maps.
-        with self.assertRaises(TypeError):
+        with self.assertRaisesRegex(TypeError,
+                                    'severity_map must be an instance'):
```

Comment:
this should be a constant, stored in the class, with an ALL_CAPS_RE variable name, like the example.